test/test-mongocrypt-marking.c

```diff
@@ -1443,6 +1535,8 @@ static void test_mc_marking_to_ciphertext_fle2_text_search(_mongocrypt_tester_t
         _mongocrypt_ciphertext_t ciphertext;
         _mongocrypt_ciphertext_init(&ciphertext);
         mongocrypt_t *crypt = _mongocrypt_tester_mongocrypt(TESTER_MONGOCRYPT_DEFAULT);
+        text_search_expected_token_counts counts = {0};
+        counts.prefixes = calculate_expected_nfix_tag_count(1, 100, 10);
```

Comment:
Though `v` is an empty string, is beta 1 to account for the `0xFF` added later?

Consider adding 1 to other `calculate_expected_nfix_tag_count` to account for the `0xFF` (though I expect is not strictly needed due to CBC padding).

Reply:
We explicitly define beta as being 1 for empty strings, as having beta = 0 would mean we generate no tokens in StrEncode, which would leak that the string is the empty string.